src/services/lnd/addInvoiceReq.ts

```diff
@@ -1,14 +1,14 @@
 import { OpenChannelRequest, Invoice } from "../../../proto/lnd/lightning";
 
-export const AddInvoiceReq = (value: number, descriptionHash = Buffer.alloc(0), expiry = 60 * 60, privateHints = false): Invoice => ({
+export const AddInvoiceReq = (value: number, expiry = 60 * 60, privateHints = false, memo?: string): Invoice => ({
```

Comment:
very minor, but I would put the " memo?: string" earlier in the params, so when you use it you dont need to set "false" for privateHints